### ISSUE TITLE
Name it just `ctpp`?

### ISSUE THREAD

**Issue by josiekre** [gregmacfarlane/ctpp_flows#1]
Thoughts on making this more general to potentially include other CTPP tables? I don't use them personally (just flows usually), but others might want to park them in the same place here. It's up for discussion but I thought I'd head off the discussion here.

**Comment by gregmacfarlane** [comment]
I like the idea of putting more tables in here and making it a one-stop shop. But these tables are pretty big, and I think people won't like packages that take an hour to download!

**Comment by gregmacfarlane** [comment]
So did you know that you can get the flows by residential characteristics? This is of course what an API would be useful for.

**Comment by gregmacfarlane** [comment]
Maybe this will have an API...

> County-to-county Worker Flow Table Based On 2009-2013 Five-Year ACS
> Census Bureau, independently of the AASHTO's CTPP program, will be producing a county-to-county commuting flow file for the 2009 to 2013 five-year ACS. It will include total flows and a short list of travel modes: drive alone, carpool, transit, and other. It is expected to be available approximately May of 2015. Census Bureau is also making progress in producing the relevant transportation tables as part of their standard products.

From [FHWA website](http://www.fhwa.dot.gov/planning/census_issues/ctpp/status_report/sr1214/)

**Comment by josiekre** [comment]
That's great news! When I was on the peer-exchange for CTPP, we really pushed the need for an API. And we also discussed the fact that most people really only use the flows and that's it. If Census could just produce that one table with their standard products, we really wouldn't need CTPP anymore.

Also, good point about the download time. The lazy load feature doesn't remove the fact that install the package would take forever still.

I say we keep this with just the flow table for now and see if Census puts it into their API. If so, this package needs to become an interface to the API. You've seen the function I already started. It needs much improvement.

I wonder how the two tables (AASHTO's version and Census' version) will differ. They are both from the same dataset technically. Right?

**Comment by josiekre** [comment]
I just got some questions from Guy Rousseau on how we should have the CTPP API specs. So it looks like it's happening!

Also, I just discovered the `UScensus2010` r package. I think it's a really good model for how to provide a useful package that only installs data you want. Check it out.

**Comment by gregmacfarlane** [comment]
I'm glad people like Guy know to talk to people like you. 

Also, the `UScensus2010` package looks super interesting. We might want to wrap this Zack Almquist guy in to help us. Though it seems like the choice of helper functions is a little arbitrary.

**Comment by josiekre** [comment]
Agreed. They are arbitrary and I'm having difficulty figuring out how to
use the package. I've realized that vignettes are really the only way to
explain how to use packages effectively.
